marshal/claim.go

```diff
@@ -42,13 +42,14 @@ type claim struct {
 	messagesLock  *sync.Mutex
 	offsets       PartitionOffsets
 	marshal       *Marshaler
+	consumer      *Consumer
```

Comment:
This up-and-down-the-stack dependency might be easier to manage going forward if you define an interface for Claim to take here instead of requiring the full-blown Consumer just to get one function? But not necessary.